content/c/concepts/basic-output/basic-output

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Basic Output"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Print"
+  - "Output"
+---
+
+In the C language basic console output is achieved using functions from the standard input output header file (stdio.h)
+
+## Including Output functions
+
+C itself has no ability to output any information, in order to so the stdio header file must be included in the program, which is typically done at the top of the file.
+
+```c
+#include <stdio.h>
+```
+
+Once this is done the functions included in it can be used.
+
+## Printing Single Characters With `putchar`
+
+The `putchar` function accepts a single `char` input, and prints it to the `stdout` file, normally the console.
+
+```pseudo
+#include <stdio.h>
+
+int main(void) {
+  putchar('a');
+  return 0;
+}
+```
+
+## Printing Strings With `puts`
+
+The `puts` function, or "put string", takes a string as an argument and prints it to the `stdout` file followed by a newline character.
+
+```c
+#include <stdio.h>
+
+int main(void) {
+  puts("Hello World!!");
+  return 0;
+}
+```
+
+## Formatting Variables With `printf`
+
+The `puts` function can only accept and print a single string argument, which is why there is the `printf` function, or "print formatted". As arguments it takes a string, containing information on what values to format, followed by the values to format in the string.
+
+```c
+#include <stdio.h>
+
+int main(void) {
+  printf("%d is a number\n", 5); // prints 5 is a number
```

Comment:
Let's move the output to it's own code block here:

The output would be:

```shell
5 is a number
2 is after 1
```

Reply:
Done